monitoring/grafana/update_risingwave_dashboard.py

```diff
@@ -0,0 +1,169 @@
+    
+output_file = "./risingwave-dashboard.json"
+if len(sys.argv) == 3:
+    output_file = sys.argv[2]
+
+url = "https://raw.githubusercontent.com/risingwavelabs/risingwave/{commit}/grafana/risingwave-dashboard.json"
+response = requests.get(url.format(commit = commit_id))
+content = response.content.decode("utf-8")
+json_data = json.loads(content)
+
+# update ["annotations"]["list"]
+annotations_list = [{
+    "builtIn": 1,
+    "datasource": {
+        "type": "grafana",
+        "uid": "-- Grafana --"
+    },
+    "enable": True,
+    "hide": True,
+    "iconColor": "rgba(0, 211, 255, 1)",
+    "name": "Annotations & Alerts",
+    "target": {
+        "limit": 100,
+        "matchAny": False,
+        "tags": [],
+        "type": "dashboard"
+    },
+    "type": "dashboard"
+}]
+templating_list = [
+    {
+        "current": {
+            "selected": False,
+            "text": "default",
+            "value": "default"
+        },
+        "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus"
+        },
+        "definition": "label_values(up{risingwave_name=~\".+\"}, namespace)",
+        "description": "Kubernetes namespace.",
+        "hide": 0,
+        "includeAll": False,
+        "label": "Namespace",
+        "multi": False,
+        "name": "namespace",
+        "options": [],
+        "query": {
+            "query": "label_values(up{risingwave_name=~\".+\"}, namespace)",
+            "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": False,
+        "sort": 1,
+        "type": "query"
+    },
+    {
+        "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus"
+        },
+        "definition": "label_values(up{namespace=\"$namespace\", risingwave_name=~\".+\"}, risingwave_name)",
+        "description": "RisingWave pod.",
+        "hide": 0,
+        "includeAll": False,
+        "label": "RisingWave",
+        "multi": False,
+        "name": "instance",
+        "options": [],
+        "query": {
+            "query": "label_values(up{namespace=\"$namespace\", risingwave_name=~\".+\"}, risingwave_name)",
+            "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": False,
+        "sort": 5,
+        "type": "query"
+    }
+]
+json_data["annotations"]["list"] = annotations_list
+json_data["templating"]["list"] = templating_list
+json_data["title"] = "RisingWave Dashboard"
+json_data["time"]["from"] = "now-5m"
+del json_data["__inputs"]
+
+panels_key = "panels"
+datasource_key = "datasource"
+targets_key = "targets"
+expr_key = "expr"
+legend_format_key = "legendFormat"
+# define "datasource"
+datasource_value = {
+    "type": "prometheus",
+    "uid": "prometheus"
+}
+
+def contains_str(string, target):
+    return target in string and target + "_" not in string and "_" + target not in string
+
+def update_expr(target): 
```

Comment:
🥲 I think this one doesn't cover all the cases and is a bit complicated. But for now, if it works properly, then just let it be.

Reply:
Yes, I'm not sure whether it can cover all the cases🥲. But it may be a comfort that I have checked the generated file manually, at least all the `"expr"` has included `{namespace=\"$namespace\", risingwave_name=\"$instance\"}`.